The Aethyrblood Chronicles/sheet.json

```diff
@@ -0,0 +1,10 @@
+{
+  "html": "aethyrblood.html",
```

Comment:
I'm not sure if these things are case-sensitive, but your html & css files have capitalized names `Aethyrblood.html`, while in the `sheet.json` you wrote `aethyrblood.html`.

Reply:
Thanks Anduh
I will change that, just in case.